modules/command/dc/dc-challenge-command.js

```diff
@@ -100,8 +102,19 @@ class DCChallengeCommand extends Command {
         if (challenge == null) {
             throw new Error(`Failed to find challenge ${challenge_id}`);
         }
-
-        this.logger.info(`Failed to send challenge for object ${challenge.object_index} and block ${challenge.block_index} to DH ${challenge.dh_id}.`);
+        const errorMessage = `Failed to send challenge for object ${challenge.object_index} and block ${challenge.block_index} to DH ${challenge.dh_id}.`;
+        this.logger.info(errorMessage);
+        this.errorNotificationService.notifyWarning(
```

Comment:
When DC failed to send a challenge the severity is warning, when DH failed to answer a challenge the severity is error, and when DC gets wrong or no answer the severity is info. Can we use the same severity level for the same issues?